docs/plans/Video.md

```
@@ -35,27 +35,25 @@ Originals Bucket:
 However, videos require a few things that still images don't:
 
 - **Transcoded video**. Videos will be transcoded to a universally playable format supported by all browsers.
-- **Poster image**. A "poster" image will be extracted from the video. It'll be used as the source to generate thumbnails and detail page image.
+- **Poster image**. A "poster" image will be extracted from the video. It'll be used as the source to generate thumbnails and detail page image. This JPG is not served to browsers; rather, it's the source for thumbs and detail image. As such it will be generated with a high quality number. Use the same quality number used by the HEIC -> JPG original conversion.
 
-Each video will get a UUID. The transcoded video and poster live in the Derived bucket by UUID and versionId:
+The transcoded video and poster live in the Derived bucket by path and versionId, consistent with how images work:
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)